Makefile

```diff
@@ -519,7 +522,28 @@ release-binary: $(RELEASE_DIR)
 release-staging: ## Builds and push container images to the staging bucket.
 	REGISTRY=$(STAGING_REGISTRY) $(MAKE) docker-build-all docker-push-all release-alias-tag
 
-RELEASE_ALIAS_TAG=$(PULL_BASE_REF)
+.PHONY: release-staging-daily
+release-staging-daily: ## Builds and push container images to the staging bucket.
+	$(eval RELEASE_ALIAS_TAG := daily_$(TAG)_$(shell date +%F))
```

Comment:
Might be I'm missing something but where are we setting TAG?

Reply:
You're right, that is coming from Prow and set to `TAG` in cloudbuild.
Set it in Makefile like this now:

```
ifneq ($(GIT_TAG),)
  TAG ?= $(GIT_TAG)
else
  TAG ?= dev
endif
```